plugins/catalog-react/report-alpha.api.md

```diff
@@ -129,13 +134,80 @@ export const EntityCardBlueprint: ExtensionBlueprint<{
         {
           optional: true;
         }
+      >
+    | ConfigurableExtensionDataRef<
+        'full' | 'info' | 'peek',
+        'catalog.entity-card-area',
+        {
+          optional: true;
+        }
+      >;
+  inputs: {};
+  config: {
+    filter: string | undefined;
+    area: 'full' | 'info' | 'peek' | undefined;
+  };
+  configInput: {
+    filter?: string | undefined;
+    area?: 'full' | 'info' | 'peek' | undefined;
+  };
+  dataRefs: {
+    filterFunction: ConfigurableExtensionDataRef<
+      (entity: Entity) => boolean,
+      'catalog.entity-filter-function',
+      {}
+    >;
+    filterExpression: ConfigurableExtensionDataRef<
+      string,
+      'catalog.entity-filter-expression',
+      {}
+    >;
+    area: ConfigurableExtensionDataRef<
+      'full' | 'info' | 'peek',
+      'catalog.entity-card-area',
+      {}
+    >;
+  };
+}>;
+
+// @alpha (undocumented)
+export const EntityCardLayoutBlueprint: ExtensionBlueprint<{
+  kind: 'entity-card-layout';
+  name: undefined;
+  params: {
+    defaultFilter?: string | ((entity: Entity) => boolean) | undefined;
```

Comment:
This doesn't line up with the rest, but might be better 🤔

Leaving as a followup to align these

Reply:
Followup in https://github.com/backstage/backstage/pull/28923